edx_sga/sga.py

```diff
@@ -415,7 +415,8 @@ def enter_grade(self, request, suffix=''):
         require(self.is_course_staff())
         module = StudentModule.objects.get(pk=request.params['module_id'])
         state = json.loads(module.state)
-        score = int(request.params['grade'])
+        # Automatically convert from number (float or not), down to int
+        score = int(float(request.params['grade']))
```

Comment:
I wasn't happy with it after discussing with @pdpinch. It now shows a javascript error if they enter in a floating point number, and I reverted it back to raising an exception if they bypass the javascript check.